packages/mobile/src/components/TokenBottomSheet.tsx

```diff
@@ -111,24 +114,21 @@ function TokenBottomSheet({ isVisible, origin, onTokenSelected, onClose }: Props
           testID={'BackgroundTouchable'}
         />
       </Animated.View>
-      <Animated.View
-        style={[
-          styles.contentContainer,
-          { paddingBottom: Math.max(safeAreaInsets.bottom, Spacing.Thick24) },
-          animatedPickerPosition,
-        ]}
+      <Animated.ScrollView
```

Comment:
Is it possible to add a test for this new behavior?

Reply:
I don't see how, this is a UI thing. I imagine with an e2e test, but Detox tests are hard to write for this kind of thing I think.

I'll keep this in mind to consider when adding e2e tests for this flow.
cc @MuckT

Reply:
We can add a e2e test to ensure it's scrollable, but testing one component isn't ideal for e2e tests. Alternatively, we might be able to test this with [React Native Testing Library](https://callstack.github.io/react-native-testing-library/docs/api/#fireeventscroll-element-reacttestinstance-data-arrayany--void) which seems more appropriate if possible.